### PR TITLE
SQL editor keyboard shortcut enhancements

### LINKED ISSUE #4395: Hotkey for run query on Windows
It will be cool to be able to run query by hotkey. It makes work a bit faster.

My favorite hotkeys for run query (in SQL Workbench/J) on Windows keyboard:
* `Ctrl + Enter` (for run query)
* `Ctrl + E` (for run selected part of query)

### PR DESCRIPTION
**Pull request by tlrobinson**
Resolves #4395

* [x] add ctrl-enter support for windows
* [x] add ctrl-option-enter / cmd-option-enter to run selected portion of query

### CODE REVIEW

**Comment by tlrobinson** [comment]
@anki-code could you try this branch if you have a chance. I added both ctrl+enter (for running whole query) and ctrl+alt+enter (for running selected text).

**Comment by anki-code** [comment]
@tlrobinson It works like a charm! Thank you! 🍻

File: frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx [hunk 1]
```diff
@@ -147,6 +150,24 @@ export default class NativeQueryEditor extends Component {
         }
     }
 
+    componentWillUnmount() {
+        document.addEventListener("keydown", this.handleKeyDown);
```
Comment (kdoh) [inline frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx]: Shouldn't we be removing the event listener here?

Comment (tlrobinson) [inline frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx]: Indeed

**Review by kdoh — APPROVED** [review]

**Pull request merged**